chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -578,6 +590,7 @@ impl pallet_session::Config for Runtime {
     type SessionHandler = <SessionKeys as OpaqueKeys>::KeyTypeIdProviders;
     type Keys = SessionKeys;
     type WeightInfo = weights::pallet_session::SubstrateWeight<Runtime>;
+    type DisablingStrategy = (); // TODO: revisit this
```

Comment:
This one is weird, it is a config that used to be part of pallet-staking, now is in pallet-session

We don't use the pallet session from parity, so not sure if we need to use the same value as them (UpToLimitWithReEnablingDisablingStrategy) or just ().

https://github.com/paritytech/polkadot-sdk/blob/7aae05ddae81e05c63778a49125f83b716c0e875/substrate/bin/node/runtime/src/lib.rs#L762

https://github.com/paritytech/polkadot-sdk/pull/7581

Reply:
we do use pallet session from parity. but yeah for now I would just put ti to (). it seems to be an improvement over something they already had. for now let's leave it as is

Reply:
Sorry I meant

> We don't use the pallet **staking** from parity